src/main/resources/templates/fragments/selected-grouping.html

```diff
@@ -89,7 +89,7 @@ <h2 ng-if="selectedGrouping.name" class="card-title text-light mb-0"> {{ selecte
                                            aria-label="Grouping Description"
                                            th:title="#{screen.message.admin.selectedGrouping.descriptionFormTitle}"
                                            th:placeholder="#{screen.message.admin.selectedGrouping.descriptionPlaceholder}"
-                                           ng-value="descriptionDisplay()"
```

Comment:
could this $scope.groupingDescription variable be used every where in place of the $scope.descriptionDisplay()?

Reply:
We still need it for groupings that have no description when displaying the description in the groupings list. I removed it from the starting value in the form because it was annoying deleting the entire "No description given for this Grouping." message over and over again. Also, when editing the form it puts in the actual value instead of the default placeholder..